pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
+line-length = 120
```

Comment:
[black_compatibility for isort](https://pycqa.github.io/isort/docs/configuration/black_compatibility.html#:~:text=To%20use%20the%20profile%20option%20when%20calling%20isort,of%20how%20this%20would%20look%20like%20in%20your.travis.yml)
```
[tool.isort]
profile = "black"
```